frontend/app/(footer)/dashboard/page.tsx

```diff
@@ -11,18 +11,8 @@ import { useRouter, useSearchParams } from "next/navigation";
 import { useEffect, useLayoutEffect, useState } from "react";
 
 export default function Dashboard() {
-  const [loading, setLoading] = useState(true);
+  const [loading, setLoading] = useState<boolean>(true);
   const [allEventsDataList, setAllEventsDataList] = useState<EventData[]>([]);
-  const loadingEventDataList: EventData[] = [
```

Comment:
might need to leave this one in as when you go from 1 or 0 events to more events, only 1 card is showed to load

Reply:
This will be a temp fix until we come back and fix the search